### PR TITLE
Fix Max Preview Size (New & Open Project)

### PR DESCRIPTION
**Pull request by jonoomph**
When opening or creating a project, we lose our timeline max-size optimization. This causes slow downs in preview and some strange visual artifacts, such as text-size changing when displaying frame #'s on a clip.

### CODE REVIEW

File: src/windows/video_widget.py [hunk 1]
```diff
@@ -509,15 +509,23 @@ def paintEvent(self, event, *args):
     def centeredViewport(self, width, height):
         """ Calculate size of viewport to maintain aspect ratio """
 
-        window_size = QSizeF(width, height)
-        window_rect = QRectF(QPointF(0, 0), window_size)
+        # Calculate padding
```
Comment (jonoomph) [inline src/windows/video_widget.py:512]: Not sure why these changes are showing different than develop (since I already merged them)

File: src/windows/video_widget.py [hunk 2]
```diff
@@ -1265,8 +1273,12 @@ def keyFrameTransformTriggered(self, effect_id, clip_id):
 
     def regionTriggered(self, clip_id):
         """Handle the 'select region' signal when it's emitted"""
-        # Clear transform
-        self.region_enabled = bool(not clip_id)
+        if self and not clip_id:
```
Comment (jonoomph) [inline src/windows/video_widget.py:1276]: Ditto

**Comment by jonoomph** [comment]
Merging to get the builders working on this one

File: src/windows/video_widget.py (hunk 1 above)
Comment (JacksonRG) [reply src/windows/video_widget.py:512]: :thinking:

**Review by JacksonRG — APPROVED** [review]
LGTM
